package.json

```diff
@@ -7,15 +7,15 @@
     "test": "jasmine"
   },
   "dependencies": {
-    "cookie-parser": "~1.4.3",
```

Comment:
Is there a reason all these dependencies are being upgraded? (It's safer to not, in case something gets broken accidentally)

Reply:
yeah they shouldn't be -- that was a mistake